index.js

```diff
@@ -212,61 +125,73 @@ export default class OneSignal {
             }
             RNOneSignal.requestPermissions(requestedPermissions);
         } else {
-            console.log("This function is not supported on Android");
+            console.log("requestPermissions: this function is not supported on Android");
         }
     }
 
-    /* deprecated */
-    static configure() {
-        console.warn("OneSignal: the `configure` method has been deprecated. The `ids` event is now triggered automatically.");
-    }
-
-    static init(appId, iOSSettings) {
-        if (!checkIfInitialized()) return;
-
+    static promptForPushNotificationPermissions(handler) {
```

Comment:
Remove `promptForPushNotificationPermissions` method, it does the same thing as `promptForPushNotificationsWithUserResponse`

Reply:
Done